src/__tests__/utils.js

```diff
@@ -30,8 +30,10 @@ type TestResponse = {
 };
 
 export class TestRequestFactory implements RequestFactory<TestRequest> {
+  baseURL: string;
+  version: string;
   createRequest = (uri: string, method?: string, body?: Object) => ({
-    uri,
+    uri: `http://localhost:4001/api/beta/${uri}`,
```

Comment:
Hm. The fact that this request factory also needs to do this work is a bit of a red flag for me, in that it'll be easy to forget to do this, even with the new interface properties on `RequestFactory`. It feels like the implementation is probably the same by default for most request factories... 🤔 

What do you think about also adding `prefixURI to `interface RequestFactory`? Perhaps that may single to other implementors of `RequestFactory` that this work still needs to be done.

What other ideas do you have around this?

Reply:
Yep, I am/was not a fan either. I tried the approach you mentionned (or at least what I interpreted). Let me know what you think.